group-actors/src/main/java/org/sunbird/actors/CacheActor.java

```diff
@@ -0,0 +1,63 @@
+package org.sunbird.actors;
+
+import java.util.Arrays;
+import java.util.Map;
+import org.apache.commons.lang3.StringUtils;
+import org.sunbird.actor.core.ActorConfig;
+import org.sunbird.cache.impl.RedisCache;
+import org.sunbird.request.Request;
+import org.sunbird.response.Response;
+import org.sunbird.util.JsonKey;
+import scala.collection.JavaConverters;
+
+@ActorConfig(
+  tasks = {"getCache", "setCache", "delCache"},
+  dispatcher = "",
+  asyncTasks = {}
+)
+public class CacheActor extends BaseActor {
+
+  private int ttl =
+      StringUtils.isNotEmpty(System.getenv("groups_redis_ttl"))
+          ? Integer.parseInt(System.getenv("groups_redis_ttl"))
+          : 3600000;
+
+  @Override
+  public void onReceive(Request request) throws Throwable {
+    String operation = request.getOperation();
+    switch (operation) {
+      case "setCache":
+        setCache(request);
+        break;
+      case "getCache":
+        getCache(request);
+        break;
+      case "delCache":
+        delCache(request);
+      default:
+        onReceiveUnsupportedMessage("CacheActor");
+    }
+  }
+
+  private void setCache(Request request) {
```

Comment:
All these methods have a log line what id they're trying with.

Reply:
When I call this Actor, I am setting reqId in the header of the request object


`[2020-07-16 17:50:07,269] [INFO] org.sunbird.actors.CacheActor:92 [] [thisActorSystem-akka.actor.default-dispatcher-5] [e8c689b4-10f9-4633-b321-6ffb1786a22b] - getCache started at 1594902007265 
[2020-07-16 17:50:07,271] [INFO] org.sunbird.actors.CacheActor:92 [] [thisActorSystem-akka.actor.default-dispatcher-5] [e8c689b4-10f9-4633-b321-6ffb1786a22b] - getCache method call 52d1ae62-42b0-4242-b271-43ba702b9f5a 
[2020-07-16 17:50:07,271] [INFO] org.sunbird.actors.CacheActor:92 [] [thisActorSystem-akka.actor.default-dispatcher-5] [e8c689b4-10f9-4633-b321-6ffb1786a22b] - getCache ended at 1594902007266 
`